File: CodeWars/Javascript/8kyu/SurfaceAreaAndAVolumeOfABox.js
function getSize(width, height, depth) {
    var arr = new Array(2);
    arr[0] = 2 * (height * width) + 2 * (height * depth) + 2 * (width * depth)
    arr[1] = width * height * depth;
    return arr;
}


/*Description:
    Write a
function that returns the total surface area and volume of a box as an array: [area, volume].*/


const getSize = (w, h, d) => [(w * h + w * d + d * h) * 2, w * h * d]

File: CodeWars/Ruby/8kyu/SurfaceAreaAndAVolumeOfABox.rb
def get_size(w,h,d)
  arr = Array.new
  arr[0] =(h*w+h*d+d*w)*2
  arr[1] = h*d*w
  arr
end

def get_size(w,h,d)
   [((2*w*d+(2*d*h)+(2*h*w))),(w*h*d)]
end

=begin
Description:

Write a function that returns the total surface area and volume of a box as an array: [area, volume].
=end
